Dependency issue when building docs according to instructions

#### Code Sample, a copy-pastable example if possible

I was following the instructions to build the docs [locally](https://xarray.pydata.org/en/latest/contributing.html#how-to-build-the-xarray-documentation), using `xarray/ci/requirements-py36` as test environment.

When running `make html` in the `xarray/doc` folder, I get this:
```
xarray: 0.12.1+9.gfad6d624.dirty, /Users/juliusbusecke/Work/CODE/PYTHON/xarray/xarray/__init__.py

Extension error:
Could not import extension IPython.sphinxext.ipython_directive (exception: No module named 'IPython')
make: *** [html] Error 2
(test_env)
```
After installing ipython with `conda install ipython`, the build works properly. 

Should this line `conda install -c conda-forge sphinx sphinx_rtd_theme sphinx-gallery numpydoc` be updated to:
`conda install -c conda-forge sphinx sphinx_rtd_theme sphinx-gallery numpydoc ipython` ?

Or am I overlooking something?

--- a/doc/contributing.rst
+++ b/doc/contributing.rst
@@ -285,17 +285,23 @@ How to build the *xarray* documentation
 
 Requirements
 ~~~~~~~~~~~~
+Make sure to follow the instructions on :ref:`creating a development environment above <contributing.dev_env>`, but
+to build the docs you need to use the environment file ``doc/environment.yml``.
 
-First, you need to have a development environment to be able to build xarray
-(see the docs on :ref:`creating a development environment above <contributing.dev_env>`).
+.. code-block:: none
+
+    # Create and activate the docs environment
+    conda env create -f doc/environment.yml
+    conda activate xarray-docs
+
+    # or with older versions of Anaconda:
+    source activate xarray-docs
+
+    # Build and install xarray
+    pip install -e .
 
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-In your development environment, install ``sphinx``, ``sphinx_rtd_theme``,
-``sphinx-gallery`` and ``numpydoc``::
-
-    conda install -c conda-forge sphinx sphinx_rtd_theme sphinx-gallery numpydoc
 
 Navigate to your local ``xarray/doc/`` directory in the console and run::
 

